server/Controllers/ideaController.js

```diff
@@ -5,117 +5,142 @@ const ideaController = {};
 
 // middleware to get all ideas data from database
 ideaController.getIdeas = (req, res, next) => {
-	const queryText = 'SELECT * FROM Ideas';
+  const queryText = 'SELECT * FROM Ideas';
 
-	model.query(queryText, (err, results) => {
-		if (err) {
-			console.log(err);
-			return next({
-				log: `error occurred at getIdeas middleware. error message is: ${err}`,
-				status: 400,
-				message: { err: 'An error occurred' },
-			});
-		}
-		// console.log('results', results.rows);
-		res.locals.ideas = results.rows;
-		return next();
-	});
+  model.query(queryText, (err, results) => {
+    if (err) {
+      console.log(err);
+      return next({
+        log: `error occurred at getIdeas middleware. error message is: ${err}`,
+        status: 400,
+        message: { err: 'An error occurred' },
+      });
+    }
+    // console.log('results', results.rows);
+    res.locals.ideas = results.rows;
+    return next();
+  });
```

Comment:
Why do all these spaces keep getting changed lol. We should all standardize the same linting configuration so our PRs don't get bloated with whitespace changes.